srcpkgs/cross-i686-linux-musl/files/bug90756.patch

```diff
@@ -0,0 +1 @@
+../../gcc/patches/bug90756.patch
```

Comment:
This needs to be applied in the template. There is no auto patching in `cross-*`.

Reply:
Oops, fixed.